### DISCUSSION TITLE
Issue Certificate Batch

### DISCUSSION THREAD

**Discussion by ygengyeng** [OpenCerts/opencerts-website#643] [General]
Hi 
I have some enquiries on https://admin.opencerts.io/

Is the certificate store address linked to the wallet account holder? Meaning only the account holder who deployed the instance can issue certificate batch using that particular certificate store address?

If wallet account holder loses access to the account and create a new account, he/she will not be able to issue certificate batch with the previous certificate store address?

Thanks

**Comment by HJunyuan** [comment]
Only the wallet owner that deployed the document store can issue certificates.

Alternatively, you may also transfer ownership of the document store (smart contract) but you will still need access to the wallet credentials.

**Reply by ygengygeng** [reply]
@HJunyuan Can the registry be updated to include additional certificate store for a training provider?
https://www.opencerts.io/static/registry.json

**Reply by Nebulis** [reply]
@ygengygeng Please use opencerts v2
